core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/DatabricksParseHelper.scala

```diff
@@ -48,6 +52,11 @@ object DatabricksParseHelper extends Logging {
   val SUB_PROP_JOB_ID = "JobId"
   val SUB_PROP_RUN_NAME = "RunName"
 
+  // Labels for Photon metrics that
```

Comment:
nit: comment is cut off?

Reply:
Thanks @cindyyuanjiang. Fixed the comments